students/rhamersky/Lesson_3/mailroom_part1.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python3
+#  Description: This program will allow...........
+# Developer: Ryan Hamersky
+# Date: 05/10/2018
+# Rev: A - 05/15/2018 updated tup_donor_names so it would fit on one page and changed 'is' to '==' when comparing the
+#      variable intChoice.
+
+# -----Data Section-----
+strName = None     # --> User's name
+intOption = int()  # --> User's option.
```

Comment:
make sure to use snake_case var names

Reply:
Corrected in Lesson 4 and Lesson 5 of the vars that I saw.